library/data/component/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Component Attachment API"
```

Comment:
Hmm, the divergence between the mod name (Quilt Component Attachment API) and the mod ID (quilt_component) is a bit weird here, and thinking about it, "Component Attachment" doesn't really make sense compared to "Registry Entry Attachments", since for the latter, registry attachments basically attach values to their registry entries, while components aren't the thing being attached on